config.rb

```diff
@@ -36,4 +36,12 @@
 #   #       language: rust
 #   #     ruby_project:
 #   #       language ruby
+
+# # You may also define additional repositories programatically as follows.
+# # These will be added to the repositories defined in projects.yml.
+# # This may be useful, for instance, if you want to use environment variables to keep information secret.
+# # Note: you need to require './lib/cutting_edge/repo.rb' first
+#   REPOSITORIES = {
+#     'gitlab/sarantine/cmb' => GitlabRepo.new(ENV['PRIVATE_REPO1_ORG'], ENV['PRIVATE_REPO1_NAME'], 'python', nil, nil, nil, ENV['PRIVATE_REPO1_TOKEN'], true)
```

Comment:
Should *this* private repo be publically advertised? 😕

Reply:
Aaargh I censored it in the `.new` call but totally overlooked the key/String I was using for testing purposes. :-( Much sorry